assets/post_install_macOS.sh

```diff
@@ -28,22 +31,23 @@ else
     APP_DIR="$HOME"/Applications
     PERMS=""
 fi
-SPI_APP_DIR="${APP_DIR}/Scientific-Python"
-logger -p 'install.info' "📓 SPI_APP_DIR=$SPI_APP_DIR"
+PKG_APP_DIR="${APP_DIR}/${INSTALLER_NAME}"
+logger -p 'install.info' "📓 PKG_APP_DIR=$PKG_APP_DIR"
 
-logger -p 'install.info' "ℹ️ Moving root SP .app bundles from $APP_DIR to $SPI_APP_DIR."
-$PERMS mv "$APP_DIR"/Scientific\ Python\ *.app "$SPI_APP_DIR"/
+logger -p 'install.info' "ℹ️ Moving root project .app bundles from $APP_DIR to $PKG_APP_DIR."
+# Set this to match the names generated from the menu package.
+$PERMS mv "${APP_DIR}"/Scientific\ Python\ *.app "$PKG_APP_DIR"/
 
-logger -p 'install.info' "ℹ️ Fixing permissions of SP .app bundles in $SPI_APP_DIR: new owner will be ${USER_FROM_HOMEDIR}."
-$PERMS chown -R "$USER_FROM_HOMEDIR" "$SPI_APP_DIR"
+logger -p 'install.info' "ℹ️ Fixing permissions of project .app bundles in $PKG_APP_DIR: new owner will be ${USER_FROM_HOMEDIR}."
+$PERMS chown -R "$USER_FROM_HOMEDIR" "$PKG_APP_DIR"
 
-SPI_ICON_PATH="${PREFIX}/Menu/spi_mac_folder_icon.png"
-logger -p 'install.info' "ℹ️ Setting custom folder icon for $SPI_APP_DIR and $SPI_APP_DIR_ROOT to $SPI_ICON_PATH."
-for destPath in "$SPI_APP_DIR" "$SPI_APP_DIR_ROOT"; do
-    logger -p 'install.info' "ℹ️ Setting custom folder icon for $destPath to $SPI_ICON_PATH."
+PKG_ICON_PATH="${PREFIX}/Menu/spi_mac_folder_icon.png"
```

Comment:
I wonder if we should genericize the image names too, to remove the `spi_` prefix (maybe in a follow-up PR?)